src/main/java/org/folio/sidecar/service/TenantService.java

```diff
@@ -28,6 +28,9 @@
 @RequiredArgsConstructor
 public class TenantService {
 
+  @ConfigProperty(name = "tenant-service.reset-task.cron-definition")
+  String resetTaskCronDefinition;
```

Comment:
where is the field used apart log message?

Reply:
@Scheduled(cron =
I